packages/@aws-cdk/aws-events/lib/schedule.ts

```diff
@@ -31,6 +31,9 @@ export abstract class Schedule {
     if (duration.toSeconds() === 0) {
       throw new Error('Duration cannot be 0');
     }
+    if (duration.toSeconds() % 60 !== 0) {
+      throw new Error(`'Duration must be a whole number of minutes, Duration provided was ${duration.toMinutes()} minutes'`);
+    }
```

Comment:
Why this? You didn't really explain the motivation for the change, but as far as I can tell on line `40` below we do `duration.toMinutes({ integral: true })`, which will throw if the duration is not convertible to a whole number of minutes.

You just added a different exception that will throw if the duration is not convertible to a whole number of minutes.

It looks like you're trading out one error for another, so: why?

- Is the original error message not good enough? If so: is that maybe something we should be clearing up instead? Can we/should we change the original error message?
  - If that's your thought process, and you considered and rejected updating the error produced by `Duration.toMinutes()`, then that would be helpful to include in your PR description, because it would give me insight into things you have considered and rejected, and will help me understand why you came up with the solution you came up with.

Reply:
The original issue asks for a change to the documentation (to explain that the duration has to be a whole number of minutes, because this was apparently a surprise to the reporter), not for a code change.

Was it a conscious decision to forego the documentation update? Did you misread the original bug report?